src/main/java/org/folio/circulation/domain/notice/session/PatronActionType.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.circulation.domain.notice.session;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum PatronActionType {
+
+  CHECK_OUT("Check-out"),
+  CHECK_IN("Check-in");
```

Comment:
Is there any behaviour in mod-circulation for check in sessions yet?

Reply:
No, there isn't.